src/Compatibility/Maps/src/iOS/MapRenderer.cs

```diff
@@ -161,8 +162,8 @@ protected override void OnElementChanged(ElementChangedEventArgs<View> e)
 				}
 
 				MessagingCenter.Subscribe<Map, MapSpan>(this, MoveMessageName, (s, a) => MoveToRegion(a), mapModel);
-				if (mapModel.LastMoveToRegion != null)
-					MoveToRegion(mapModel.LastMoveToRegion, false);
+				//if (mapModel.LastMoveToRegion != null)
```

Comment:
Intentionally left commented?

Reply:
Yes we removed this api, but i want to leave the code there if we need to check something on the "old renderer"